tree.yaml

```
@@ -372,3 +372,5 @@ raw:
     radiosondes: QmPek1vjzTrhv3j7jFeSBYZz9LqCS8dvBKJyFTXgwpQFNk
     sunphotometer: QmbysfGWPM2b15X3CzTdPS9zHLyRTauwFunTAtGo8Zmviw
     WindLidar-Abacus: QmP8hH55iceYnrwoHctDtcBy6DjChqhM9shL5zjstcr6Wf
+  GENERAL:
+    campaign_logos: QmRUTSbHGpkMKze5w1kjpAc63Q8yVsZcgRX8CQ7FX38Rr7
```

Do you really want to create this as a subdirectory of `raw/`?

This seems more like something one could add on the top-level:

```suggestion
assets:
  campaign_logos: QmRUTSbHGpkMKze5w1kjpAc63Q8yVsZcgRX8CQ7FX38Rr7
```

---

I wasn't sure where it fits best. But if you think, top-level is more suitable, let's go for top-level 👍